src/mixins/panel.js

```diff
@@ -39,7 +39,7 @@ export const createPanelProviderMixin = () => ({
   },
 });
 
-export const createPanelConsumerMixin = ({ mapStateToCanRefine }) => ({
+export const createPanelConsumerMixin = () => ({
```

Comment:
```suggestion
export const createPanelConsumerMixin = ({ mapStateToCanRefine = state => Boolean(state.canRefine) }) => ({
```

Reply:
this way external usage doesn't experience a breaking change :)

Reply:
Oh indeed, didn't see it was exported to users !